CHANGELOG.md

```diff
@@ -1,3 +1,72 @@
+
+![The games section showing Cyberpunk on Steam and GOG as options.](./docs/changelog-assets/80c1fdba8263069e9bbe9f640dca8b31.webp)
+
+The app offers the following features:
+
+* Compatibility with the Steam and GOG Galaxy releases on Windows and Linux.
+* Support for the most common types of mod: REDmod, archives, REDscript, and framework mods (e.g. Cyber Engine Tweaks).
+* Support for collections.
+* Health Check diagnostics for missing dependencies.
+* Health Check diagnostics to help set up a Linux environment to mod the game.
+* REDmod Load Order management and deployment.
+
+You can read more about the available features in the [documentation](https://nexus-mods.github.io/NexusMods.App/users/games/Cyberpunk2077/). 
+
+## New Load Order Tab
+To make it easier to access your REDmod Load Order in Cyberpunk 2077, we've added a new tab to the "My Mods" and "All" sections in the left menu. 
+
+![An animation showing REDmod load order entries being dragged to new positions and bumped up and down using the arrow buttons.](https://staticdelivery.nexusmods.com/mods/2295/images/26/26-1747234539-292870229.gif)
+
+When installing mods without using collections, the "My Mods" screen will allow you manage your load order via the "Rules" tab.
+
+When using multiple collections, the "All" section under "Installed Mods" will allow you to manage your load order via the "Rules" tab. 
+
+This tab will be expanded in future to include other load-order or file-order rules related to the mods in your loadout or collection. 
+
+## More Features
+* Updated the Health Check messages for Cyberpunk 2077 to match the structure of other diagnostic messages.
+* When hovering the cursor to resize panels, a solid line will now appear to better indicate which panel will be adjusted.
+* The app now has its own minimise, maximise and close buttons, removing the operating system "chrome" and saving more vertical space. 
+* The tab bar is now hidden until there are at least two tabs in a panel.
+* Tabs can now be closed with a middle click.
+* Added mod thumbnails to the load order page.
+* Increased resilience to network errors during downloads.
+
+## Technical Changes
+* On Linux, the app will only register itself as the default Nexus Mods download handler when not installed with a package manager.
+* Replaced the Stardew Valley preview UI elements with game-agnostic versions.
+* Added the core backend frameworks required for the "Undo" feature.
+* Added the core backend frameworks to allow the app to create and share collections.
+* Improved the stability of downloads by requesting the entire file rather than using range requests.
+* Use the recent rating for collections instead of "All time".
+* The app will now send headers with Nexus Mods requests to allow us to better track mod manager usage.
+
+## Bug Fixes
+* Games can be unmanaged via the command line (thanks to Michael-Kowata on GitHub).
```

Comment:
You can tag people with @ , it will actually work when displayed in GitHub.

Reply:
GitHub isn't the primary place this will be shown so I'm not worried about that.